config/locales/en.yml

```diff
@@ -140,6 +140,10 @@ en:
           attributes:
             aso_new_headteacher:
               blank: Choose whether or not you are in your first 2 years of headship
+        forms/kind_of_nursery:
+          attributes:
+            kind_of_nursery:
+              blank: Choose what work setting you work in
```

Comment:
I added this as appropriate wording but maybe not needed? @Lockyy

Reply:
Worth running by Emma/Diana I think, should be good but worth getting a thumbs up.

@EmmaFrith @Diana-Mundo